themes/openy_themes/openy_rose/templates/page/page.html.twig

```diff
@@ -126,7 +126,11 @@
 
     {{ page.header }}
 
-    {{ page.breadcrumb }}
+    <div class="breadcrumbs">
```

Comment:
we should not print region wrappers in case there is no content inside. I would recommend you move these wrappers into block template. You can find an example in the `classy` theme from the core.